app/controllers/items_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ItemsController < ApplicationController
+  def index 
+    @items = ItemFacade.get_items
+  end 
+
+  def show 
+    @item = ItemFacade.get_item(params[:id].to_i)
```

Comment:
I don't think you'll need to turn this into an integer here since you'll be interpolating it into a request to your BE API.